tst/RtcpFunctionalityTest.cpp

```diff
@@ -175,6 +175,30 @@ TEST_F(RtcpFunctionalityTest, rembValueGet) {
     EXPECT_EQ(STATUS_RTCP_INPUT_REMB_INVALID, rembValueGet(invalidSSRCLength, SIZEOF(invalidSSRCLength), &maximumBitRate, ssrcList, &ssrcListLen));
 }
 
+static void on_PLI_callback(UINT64 opaque) {
+    *(PBOOL) opaque = TRUE;
+}
+
+TEST_F(RtcpFunctionalityTest, onpli) {
+    initializeEndianness();
+    BYTE rawRtcpPacket[] = {0x81, 0xCE, 0x00, 0x02, 0x00, 0x00, 0x00, 0x01, 0x1D, 0xC8, 0x69, 0x91};
+    RtcpPacket rtcpPacket = {0};
+    EXPECT_EQ(STATUS_SUCCESS, setRtcpPacketFromBytes(rawRtcpPacket, SIZEOF(rawRtcpPacket), &rtcpPacket));
+    ASSERT_TRUE(rtcpPacket.header.packetType == RTCP_PACKET_TYPE_PAYLOAD_SPECIFIC_FEEDBACK &&
+                rtcpPacket.header.receptionReportCount == RTCP_PSFB_PLI);
+    KvsPeerConnection kpc = {0};
+    doubleListCreate(&kpc.pTransceievers);
+    KvsRtpTransceiver kvsRtpTransceiver;
+    MEMSET(&kvsRtpTransceiver, 0, sizeof(KvsRtpTransceiver));
+    kvsRtpTransceiver.sender.ssrc =  0x1DC86991;
+    BOOL on_picture_loss_called = FALSE;
+    kvsRtpTransceiver.onPictureLossCustomData = (UINT64) &on_picture_loss_called;
+    kvsRtpTransceiver.onPictureLoss = on_PLI_callback;
+    doubleListInsertItemHead(kpc.pTransceievers, (UINT64) &kvsRtpTransceiver);
+    onRtcpPLIPacket(&rtcpPacket, &kpc);
```

Comment:
Mind passing this as a lambda? Just so we don't have to add another function that isn't tightly tied to the test

Reply:
done